pkg/utils/yaml.go

```diff
@@ -30,7 +30,7 @@ func SplitYamlDocuments(fileBytes Document) ([]Document, error) {
 func IsEmptyYamlDocument(document Document) bool {
```

Comment:
Could you add some godoc explaining that this function returns true for comment-only single documents, and strings with multiple documents where all docs are comment-only.

Reply:
> comment-only single documents

But it'll work for files like this too:
1: 
```
# test doc
# test doc
---
# test doc
# test doc
---
# test doc
# test doc
```
2: 
```
---
---
```

I think gendoc should be like this:
This function validate emptiness of yaml if it only contains comment-only documents or strings with multiple documents.

If we want to make the second example invalid we should change the logic here.

Reply:
Yes those two cases are covered by the second clause of my suggestion:

> Could you add some godoc explaining that this function returns true for comment-only single documents, ***and strings with multiple documents where all docs are comment-only***.

I'm not particular about the specific wording, as long as it mentions it works on multiple document yaml files, and comments are still considered empty.

Reply:
Updated, PTAL